pkg/fakerp/shared/shared.go

```diff
@@ -1,35 +1,44 @@
 package shared
 
 import (
+	"io/ioutil"
 	"os"
-	"path/filepath"
+
+	"github.com/ghodss/yaml"
 
 	"github.com/openshift/openshift-azure/pkg/api"
-	"github.com/openshift/openshift-azure/pkg/util/managedcluster"
 )
 
 const (
-	DataDirectory = "_data/"
 	LocalHttpAddr = "localhost:8080"
 )
 
+// PkgRoot is the openshift-azure root directory
+var PkgRoot = "."
+
+func init() {
+	if gopath, found := os.LookupEnv("GOPATH"); found {
```

Comment:
I feel this will lead to instances of you doing development in a fresh clone and not seeing your tests reflected because it is being overridden by GOPATH, I'd feel like `var PkgRoot = filepath.Abs(".")` would be enough, wdyt?

Reply:
yes, this was mucked up.  I've had another go - ptal